amplify/src/main/java/com/github/stkent/amplify/prompt/CustomLayoutPromptViewConfig.java

```diff
@@ -77,8 +70,16 @@ public CustomLayoutPromptViewConfig(
         this.thanksLayout = thanksLayout;
     }
 
+    public boolean isValid() {
```

Comment:
This chain of logic was required to allow views to be instantiated from xml but configured in code.